test/integration/visitor_can_view_deal_page_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
+
+class VisitorCanViewDealPageTest < ActionDispatch::IntegrationTest
+  include CategoryItemsSetup
+  test "visitor can view the daily deal page" do
+    create_categories_and_items
+    create_featured_item
+    visit '/deal'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.